Exercice_2_-_Le_chiffre_de_Cesar/caesar.py

```diff
@@ -129,8 +129,20 @@ def create_transtab(key: int) -> str:
     return transtab
 
 def do_cipher(plain_text: str, key: int) -> str:
+    """Cipher a string object with the given key"""
+    if key > 26 or key < -26:
+        logging_print(
+            f"{Colors.RED}Need to have a key between -26 and 26{Colors.NC}",
+            logging.error,
+        )
+        sys.exit()
```

Comment:
C'est un peu brutal le `sys.exit()` en cas d'erreur. Lève pluot une exception standard genre `ValueError` vu que sa description est littéralement _Raised when an operation or function receives an argument that has the right type but an inappropriate value_.